crates/matrix-sdk-common/src/snapshots/matrix_sdk_common__deserialized_responses__tests__snapshot_test_algorithm_info.snap

```diff
@@ -0,0 +1,13 @@
+---
+source: crates/matrix-sdk-common/src/deserialized_responses.rs
+expression: "serde_json::to_value(&info).unwrap()"
+---
+{
+  "MegolmV1AesSha2": {
```

Comment:
Nice, now we don't need to imagine how things will get serialized either, we can see it right here.